asyn/drvAsynSerial/drvAsynSerialPort.c

```diff
@@ -849,9 +863,14 @@ static asynStatus writeIt(void *drvPvt, asynUser *pasynUser,
     }
     if (timerStarted) epicsTimerCancel(tty->timer);
 #ifndef vxWorks
-    if (tty->break_len > 0) {
+    if (tty->break_duration > 0) {
         tcdrain(tty->fd); /* ensure all data transmitted prior to break */
-        if (tcsendbreak(tty->fd, tty->break_len) < 0) {
+		
+		if (tty->break_delay > 0) {
+			usleep(tty->break_delay * 1000);
```

Comment:
@FreddieAkeroyd is there a better/accepted way to sleep on linux? I couldn't find another sleep call to copy so I've used http://man7.org/linux/man-pages/man3/usleep.3.html, is this ok?